afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+reprimande	-2
+irettesettelser	-2
+repulse	-1
+frastøtt	-2
+redde	2
+reddet	2
+redder	2
+ergerlig	-2
+si opp	-1
+gikk av	-1
+trekker seg	-1
+resolutt	2
+vedtak	2
+løse	2
+løst	1
+løser	1
+løse	1
+respekt	2
+respektert	2
+henseender	2
+ansvar	1
+ansvarlig	2
+mottakelig	2
+avslappende	2
+rastløs	-2
+restaurere	1
+restaurert	1
+gjenoppretting	1
+gjenopprette	1
+begrense	-2
+restriktiv	-1
+begrenser	-2
+beholdes	-1
+tilbakestående	-2
+retrett	-1
+hevngjerrig	-2
+ærede	2
+gjenopplive	2
+gjenoppliver	2
+belønning	2
+belønnet	2
+givende	2
+belønninger	2
+rik	2
+rikt	2
+rigg	-1
+rigget	-1
+riktig retning	3
+rettmessige	2
+rettmessig	2
+streng	3
+opptøyer	-2
+stige	1
+reiser seg	1
+fare	-2
+risikoer	-2
+risikabelt	-2
+medrivende	3
+rane	-2
+raner	-2
+robs	-2
+roflmao	4
+romanse	2
+kjærester	2
+rotfl	4
+råtten	-3
+uhøflig	-2
+ruinert	-2
+ruiner	-2
+sabotasje	-2
+lei seg	-2
+trist	-2
+dessverre	-2
+trygt	1
+sikrere	2
+sikkerhet	1
+honnør	2
+hilste	2
+salutter	2
+frelse	2
+sarkastisk	-2
+lagre	2
+lagret	2
+sparing	1
+skandale	-3
+skandaløs	-3
+skandalene	-3
+syndebukk	-2
+syndebukker	-2
+arr	-2
+skummelt	-2
+skeptisk	-2
+skjenne	-2
+kule	3
+hånlig	-2
+hyle	-2
+skrek	-2
+skrudd	-2
+feilet	-3
+avskum	-3
+sømløs	2
+sømløst	2
+sikre	2
+sikret	2
+oppvigleri	-2
+opprørsk	-2
+forført	-1
+selvbesmittelse	-2
+selvsikkerhet	2
+selvmotsigende	-2
+selv villedet	-2
+selvopptatt	-3
+egoisme	-3
+setning	-2
+setninger	-2
+utmålingen	-2
+bosetting	1
+bosetninger	1
+alvorlig	-2
+kjønnsdiskriminerende	-2
+skjelvende	-2
+skam	-2
+skjemmes	-2
+skammelig	-2
+dele	1
+delt	1
+aksjer	1
+sjokk	-2
+sjokkert	-2
+sjokkerende	-2
+støt	-2
+skyte	-1
+nærsynt	-2
+nærsynthet	-2
+spissmus	-4
+sjenert	-2
+syk	-2
+bivirkning	-2
+bivirkninger	-2
+sukk	-2
+betydelige	1
+lyddemping	-1
+enkelhet	1
+oppriktig	2
+vennlig hilsen	2
+since	2
+oppriktighet	2
+syndig	-3
+ensporet	-2
+synder	-2
+skeptiker	-2
+skepsis	-2
+skeptikere	-2
+skråstrek	-2
+kuttet	-2
+skråstreker	-2
+slaveri	-3
+glatt	2
+stiligere	2
+slurvet	-2
+treg	-2
+ludder	-5
+smartere	2
+smarteste	2
+smøre	-2
+smil	2
+smilte	2
+smiler	2
+smilende	2
+smugle	-2
+smuglet	-2
+smugling	-2
+smugler	-2
+sleipe	-1
+nyse	-2
+nøs	-2
+nyser	-2
+nysing	-2
+trykkrør	-2
+tankevekkende	1
+høytidelig	-1
+fast	2
+solidaritet	2
+størknet	2
+stivner	2
+størkne	2
+løsning	1
+løsninger	1
+en type	0
+son-of-a-tispe	-5
+sofistikert	2
+sår	-1
+sorgfull	-2
+beklager	-1
+romslig	1
+spammere	-3
+gnist	1
+gnisten	3
+glitrer	3
+musserende	3
+spydspiss	2
+spekulativ	-2
+ånd	1
+spirit	-2
+ondskapsfull	-2
+bortskjemte	-2
+plettfrie	2
+struttende	2
+sløse	-2
+sløst bort	-2
+sløsing	-2
+stikke	-2
+knivstukket	-2
+stabil	2
+stoppet	-1
+steile	-2
+utholdenhet	2
+stinket	-2
+oppsiktsvekkende	3
+sulte	-2
+sultet	-2
+sulter	-2
+standhaftig	2
+stjele	-2
+stjeler	-2
+stereotype	-2
+kvalt	-2
+stimulere	1
+stimulert	1
+stimulerer	1
+stimulerende	2
+stikkende	-2
+stinke	-2
+stinkende	-2
+stinker	-2
+stjal	-2
+stjålet	-2
+stoppe	-1
+stoppe	-1
+stopp	-1
+traust	2
+rett	1
+rar	-1
```

Comment:
This word seems to have a wrong sentiment

Reply:
"rar" is maybe a bit different in norwegian than in danish.
In norwegian "rar" means odd/strange, so I think the sentiment is correct.